src/uu/lsmem/src/lsmem.rs

```diff
@@ -548,4 +628,32 @@ pub fn uu_app() -> Command {
                 .help("print SIZE in bytes rather than in human readable format")
                 .action(ArgAction::SetTrue),
         )
+        .arg(
+            Arg::new(options::NOHEADINGS)
+                .short('n')
+                .long("noheadings")
+                .help("don't print headings")
+                .action(ArgAction::SetTrue),
+        )
+        .arg(
+            Arg::new(options::JSON)
+                .short('J')
+                .long("json")
+                .help("use JSON output format")
+                .action(ArgAction::SetTrue),
+        )
+        .arg(
+            Arg::new(options::PAIRS)
+                .short('P')
+                .long("pairs")
+                .help("use key=\"value\" output format")
+                .action(ArgAction::SetTrue),
+        )
+        .arg(
+            Arg::new(options::RAW)
+                .short('r')
+                .long("raw")
+                .help("use raw output format")
+                .action(ArgAction::SetTrue),
+        )
```

Comment:
These three args should conflict with each other.

Reply:
done